Ktisis/Configuration.cs

```diff
@@ -26,25 +26,25 @@ public class Configuration : IPluginConfiguration {
 		public float SkeletonLineThickness { get; set; } = 2.0F;
 		public UInt32 CategoryColor(Bone bone)
 		{
-			Vector4? color;
+			Vector4 color;
 			if (LinkBoneCategoryColors) color = LinkedBoneCategoryColor;
-			else BoneCategoryColors.TryGetValue(bone.Category.Name, out color);
-			color ??= LinkedBoneCategoryColor;
+			else if (!BoneCategoryColors.TryGetValue(bone.Category.Name, out color))
+				color = LinkedBoneCategoryColor;
 
-			return ImGui.GetColorU32((Vector4)color);
+			return ImGui.GetColorU32(color);
 		}
 		public bool IsBoneVisible(Bone bone)
 		{
-			ShowBoneByCategory.TryGetValue(bone.Category.Name, out bool? boneVisible);
-			boneVisible ??= true;
-			return DrawLinesOnSkeleton && (bool)boneVisible;
+			if (!ShowBoneByCategory.TryGetValue(bone.Category.Name, out bool boneVisible))
+				return true;
```

Comment:
This returns `true` even if `DrawLinesOnSkeleton` is `false` -- is that intentional?

Reply:
yes, this is in case a bone is uncategorizable so it shows it.
If it returned false, we would never hear of uncategorizable bones.
Maybe I should make a special category for those 🤔

Reply:
whoops, i take this back, no it's a mistake, it should return `DrawLinesOnSkeleton`